### PR TITLE
Deserialization in c#

### LINKED ISSUE #4715: Deserialization in C#
#### Topic suggestion
Deserialization in C#
#### Proposed article introduction
The accompanying article gives the Deserialization structure in C #. Allow us first to think about the course of serialization. Serialization is the method involved with changing an object into a structure over to be written in a stream and can be stored in a file, database or in a memory. Its fundamental design is to keep up with the state of the object. 
Presently, deserialization is the contrary course of serialization. It is the most common way of changing over an information stream back to an object to load to memory. This interaction empowers us to rebuild an object at whatever point needed.

#### Key takeaways 

1. Syntax and explanation
2. How deserialization works in c#.
3. Examples of deserialization in c#.

#### Article quality
This article will be different from what has been published in other blogsites. I will use simple language for anyone reading this tutorial to grasps the concepts in an easier way. I will make sure that it goes beyond other official docs and that it adds value to the rest of EngEd.
#### References
N/A

### PR DESCRIPTION
**Pull request by swambui1**
#### Formatting and structure checks
- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [X] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
This closes #4715

### CODE REVIEW

**Comment by hectorkambow** [comment]
@Dawe-7  

Lets finish up the ones you have assigned before taking on anymore 👍

**Comment by Kayere** [comment]
Plagiarism check done.

File: content/articles/deserialization-in-c#/index.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+### Introduction
+We ought to at first see what is serialization in c#. Is the most common way of bringing an object into a structure that tends to be composed in memory.
```
Comment (WanjaMIKE) [inline content/articles/deserialization-in-c#/index.md:2]: Could you avoid repetitions in sentences such as `common way`.

Comment (swambui1) [reply content/articles/deserialization-in-c#/index.md]: Hi @WanjaMIKE , I have made the changes

**Comment by Kayere** [comment]
Congratulations on publishing your first article @swambui1. Awesome job done!

Thank you for the review @WanjaMIKE 🚀.

**Pull request merged**